unyt/tests/test_array_functions.py

```diff
@@ -1838,6 +1838,12 @@ def test_unique_values():
     q = np.arange(9).reshape(3, 3) * cm
     values = np.unique(q, equal_nan=False)
     res = np.unique_values(q)
+
+    # np.unique and np.unique_values' outputs are not guaranteed to be sorted,
```

Comment:
Actually, `unique` *is* guaranteed to be sorted of course!  It isn't a (relatively) new function (not that it matters).

Reply:
wait, why does it need a `sorted` kwarg if the result is guaranteed-sorted already ?

Reply:
Yes, you can do `sorted=False`.

Reply:
oh I see, `sorted=True` is the default behavior. Thanks !